### PR TITLE
clientScripts method added

### PR DESCRIPTION
**Pull request by fixedmichal**
clientScripts method added

### CODE REVIEW

File: src/testcafe/schema.d.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ export interface TestcafeBuilderOptions {
     allowMultipleWindows?: boolean;
     assertionTimeout?: number;
     browsers?: string[]; // default in schema.json
-    clientScripts?: string | string[]; //doesn't do anything yet
+    clientScripts?: string | string[];
```
Comment (miherlosev) [inline src/testcafe/schema.d.ts:5]: Please copy the typing from [here](https://github.com/DevExpress/testcafe/blob/cde211fd164938e0db0a29c7ede17fc473d6fdf1/ts-defs-src/runner-api/runner-api.d.ts#L146).

File: src/testcafe/testcafe.builder.ts [hunk 2]
```diff
@@ -34,6 +34,7 @@ async function runTestcafe (opts: TestcafeBuilderOptions, hostName): Promise<unk
         .src(opts.src instanceof Array ? opts.src : [ opts.src ])
         .tsConfigPath(opts.tsConfigPath)
         .browsers(opts.browsers)
+        .clientScripts(opts.clientScripts instanceof Array ? opts.clientScripts : [ opts.clientScripts ])
```
Comment (miherlosev) [inline src/testcafe/testcafe.builder.ts:37]: You can simplify code as `.clientScripts(opts.clientScripts)`

**Review by miherlosev — CHANGES_REQUESTED** [review]
Thank you for your contribution to TestCafe. I left a few minor remarks.

**Comment by AndreyBelym** [comment]
I accidentally pushed the upstream `master` into the fork, so GitHub closed the PR automatically due to no differences between the PR and `master`. And since I cannot reopen it, let's continue working on this feature in #88.

**Pull request closed without merging**